ldms/python/ldmsd/ldmsd_communicator.py

```diff
@@ -226,6 +226,24 @@
                       'prdcr_listen_status': {'req_attr': [], 'opt_attr': []},
                       }
 
+def get_cmd_attr_list(cmd_verb):
+    """Return the dictionary of command attributes
+
+    If there are no required/optional attributes, the value of the
+    'req'/'opt' key is None. Otherweise, the value is a list of attribute
```

Comment:
@narategithub nit: otherwise

Reply:
Whoops. Thank you, @tom95858, for catching this.